frontend/lib/norent/components/subscribe.tsx

```diff
@@ -0,0 +1,136 @@
+import React from "react";
+import { Trans, t } from "@lingui/macro";
+import { li18n } from "../../i18n-lingui";
+import { StaticImage } from "../../ui/static-image";
+import { getImageSrc } from "../homepage";
+
+type SubscribeProps = {};
+
+type SubscribeState = {
+  email: string;
+  success: boolean;
+  response: string;
+};
+
```

Comment:
@toolness is it worth it to use the useState react hook here to condense things, and get CodeClimate to leave us alone?

Reply:
you know what, I'm just gonna do it